requirements.txt

```diff
@@ -22,6 +22,7 @@ flask==1.1.1
 idna==2.9                 # via email-validator
 itsdangerous==1.1.0       # via flask
 jinja2==2.10.1            # via flask, flask-babel
+jmespath==0.9.5
```

Comment:
If it's not on `setup.py` it can't be here, this file is auto generated by pip-compile. We could add it to requirements-dev.txt if we decide it's not a root dependency

Reply:
Thanks! I think it's not a root dependency. Added to extras.